### DISCUSSION TITLE
Please merge main into your working branch

### DISCUSSION THREAD

**Discussion by sigmafelix** [Spatiotemporal-Exposures-and-Toxicology/NRT-AP-Model#239] [Show and tell]
@dzilber @eva0marques @dawranadeep @MAKassien @mitchellmanware 

The current `main` got a lot of changes in download functions and tests. Please merge `main` at your earliest convenience into your working branch to make sure we are on the same page. Thank you!

**Comment by mitchellmanware** [comment]
I will remove branch mm_download_unit_tests_1214 as all commits have been merged to main via kpm-pkgdown-01 branc pull request.

Thank you, Insang!
